src/table/table.component.ts

```diff
@@ -30,6 +30,8 @@ export interface TableTranslations {
 	CHECKBOX_ROW: string;
 }
 
+export type TableRowSize = "sm" | "sh" | "md" | "lg";
```

Comment:
Enums are tricky since to use them properly you have to first assign them to a class property ... Hence why I've declared types like that (plus, if they do become viable, it's not a breaking change to swap them over)